pkg/framework/framework.go

```diff
@@ -41,6 +41,7 @@ var (
 	Timeout              = time.Second * 600
 	CleanupRetryInterval = time.Second * 1
 	CleanupTimeout       = time.Second * 5
+	MessagingClientsImage = "quay.io/enmasse/systemtests-clients:latest"
```

Comment:
This seems to be misplaced here. Should be put somewhere in the pkg/api/client package.